bundle/config/resources.go

```diff
@@ -25,6 +26,15 @@ type UniqueResourceIdTracker struct {
 	ConfigPath map[string]string
 }
 
+type ConfigResource interface {
+	Exists(ctx context.Context, w *databricks.WorkspaceClient, id string) (bool, error)
+	TerraformResourceName() string
+	Validate() error
+
+	json.Marshaler
+	json.Unmarshaler
```

Comment:
Why are these needed?

Reply:
This is just the campsite rule. Both of already exist in all struct implementing this interface; I'm just making them explicit.

Reply:
FYI, these are superfluous here. They implement this interface by embedding the Go SDK structs, but that is _not_ what we need. See `TestCustomMarshallerIsImplemented`.